examples/misc/multiprocess.py

```diff
@@ -79,6 +80,7 @@ def main():
         pl.plot()
         time.sleep(0.5)
     raw_input('press Enter...')
+#    input('press Enter...')    #Python3 
```

Comment:
We could also remove this line, actually it does not add anything to this example.